cluster/examples/kubernetes/ceph/object-multisite.yaml

```diff
@@ -0,0 +1,30 @@
+#################################################################################################################
+# Create an object store with settings for a test environment. Only a single OSD is required in this example.
+#  kubectl create -f object-test.yaml
+#################################################################################################################
+
+apiVersion: ceph.rook.io/v1
+kind: CephObjectStore
+metadata:
+  name: multisite-store
+  namespace: rook-ceph
+spec:
+  metadataPool:
+    failureDomain: host
+    replicated:
+      size: 3
```

Comment:
In minikube I got caught on the replica 3 default in the example. How about setting the replica to 1 to make it easier to test in the short term? Then we can change it back to 3.